pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -1088,6 +1093,7 @@ func pruneBundle(tx *gorm.DB, trustDomainID string, expiry time.Time, log logrus
 
 	// Update only if bundle was modified
 	if changed {
+		newBundle.SequenceNumber++
```

Comment:
newBundle will always come with SequenceNumber = 0, so this will reset the SequenceNumber to 1 when pruning. We should increment the number from the current bundle.

Reply:
Good catch! I fixed it and updated the prune test to use sequence numbers other than 0 and 1 to catch this.